SPECS/gettext/gettext.spec

```diff
@@ -55,7 +73,19 @@ make %{?_smp_mflags} check
 %{_datarootdir}/%{name}/*
 %{_mandir}/*
 
+%files common-devel
+%{_datadir}/%{name}/archive.*.tar.xz
+
+%files libs
```

Comment:
Please remove these from the default package now. It seems there are almost no dependencies on `gettext` at the moment, so it is safe to do that (with more dependencies, we might need to leave the duplicates or use `Provides` instead). Ideally, we could also use this opportunity to split out the `*-devel` subpackage, but one thing at a time...;)

Reply:
Sure, it makes compelete sense after our offline discussion.
Appreciate you being vigilant. Thanks